e2e-tests/Makefile

```diff
@@ -7,20 +7,20 @@ stop: stop-server stop-dashboard clean-env-placeholders
 
 .PHONY: start-server prepare-server run-cypress stop-server restart-server
 start-server:
-	./.ci/server.start.sh
+	bash ./.ci/server.start.sh
```

Comment:
What is the reason to prepend `bash` here? From [here](https://github.com/mattermost/mattermost/pull/24541/files#diff-d3a2eba57b09fd1aa50bd75274f7837d31478ba7ad18089516e703f44f6688c1R100) it looks like bash is the default shell.

Reply:
The `make` command will indeed be run using bash, due to the line you linked.

However, there's an unfortunate complication with [Makefiles and SHELL selection in windows](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html), which causes [this line to be ignored](https://github.com/mattermost/mattermost/pull/24541/files#diff-3d421e516133403ed43c4871b2ebdc8c633caf9cb7e2468b2f8868d7e1e37029R1) and as a result, Powershell is the default shell within the Makefile. Using `bash ./.ci/server.start.sh` will force powershell to invoke bash, to run the script.

Reply:
Ah okay, so it looks like we need to have a `bash.exe` file in some directory under $PATH, which it looks like we don't. Thanks for the info Mario :)